test/php/gutenberg-coding-standards/Gutenberg/Tests/AbstractSniffUnitTest.php

```diff
@@ -65,7 +67,7 @@ public function setCliValues( $filename, $config ) {
 		if ( ! isset( $GLOBALS['PHP_CODESNIFFER_RULESETS']['Gutenberg'] )
 			|| ( ! $GLOBALS['PHP_CODESNIFFER_RULESETS']['Gutenberg'] instanceof Ruleset )
 		) {
-			throw new \RuntimeException( $error_message );
+			throw new \RuntimeException( $error_message ); //phpcs:ignore WordPress.Security.EscapeOutput.ExceptionNotEscaped -- this is non-production code.
```

Comment:
Why this was not needed before and now it is needed?

Reply:
The errors likely appeared because I used `composer run check-cs` this time, instead of just `composer run lint` as I had previously done.

Reply:
Thanks for the additional details.

Just a nitpick, but I believe the recommended format for the ignore comment includes a space after the two forward slashes (https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file). So, `// phpcs:ignore` instead of `//phpcs:ignore`. The current version works so feel free to ignore this remark.

Also, not something to be addressed in this PR, but I wonder if `composer run check-cs` should be run in CI so that it is easier to catch future coding standard violations like this one?

Reply:
Good to know. Fixed in 333ccc1ce37cfd8c519a83311bbf4417e0712e29.

> Also, not something to be addressed in this PR, but I wonder if composer run check-cs should be run in CI so that it is easier to catch future coding standard violations like this one?

Yes, I thought about it, and that would be the right way to do it, in my opinion.
I just don't see it as a high-priority task since there is not a lot of development happening right now (all known issues have been addressed), and it's not likely that the GBCS package will grow significantly in the near future.